packages/bitcoin-dlc-provider/lib/BitcoinDlcProvider.ts

```diff
@@ -1475,10 +1497,14 @@ Payout Group not found',
     dlcAccept.acceptCollateralSatoshis = acceptCollateralSatoshis;
     dlcAccept.fundingPubKey = fundingPubKey;
     dlcAccept.payoutSPK = payoutSPK;
-    dlcAccept.payoutSerialId = payoutSerialId;
+    dlcAccept.payoutSerialId = TEMP_FORCE_SERIAL_ID_ORDERING
+      ? BigInt(Number(dlcOffer.payoutSerialId) + 1)
+      : (dlcAccept.payoutSerialId = payoutSerialId); // TODO SERIAL_ID remove temp force serial id ordering
     dlcAccept.fundingInputs = fundingInputs;
     dlcAccept.changeSPK = changeSPK;
-    dlcAccept.changeSerialId = changeSerialId;
+    dlcAccept.changeSerialId = TEMP_FORCE_SERIAL_ID_ORDERING
+      ? BigInt(Number(dlcOffer.changeSerialId) + 1)
+      : (dlcAccept.changeSerialId = changeSerialId); // TODO SERIAL_ID remove temp force serial id ordering
```

Comment:
Any reason we're assigning in the ternary condition?

`: (dlcAccept.changeSerialId = changeSerialId); `
`: (dlcAccept.payoutSerialId = payoutSerialId);`
`: (dlcOffer.fundOutputSerialId = fundOutputSerialId);`

Reply:
Originally implemented with `TEMP_FORCE_SERIAL_ID_ORDERING` temp global var to enable one client to be updated before another. In the latest commit this has been removed in favor of force update.